### DISCUSSION TITLE
Can I get source code of @Value internal implement spring

### DISCUSSION THREAD

**Discussion by arvindersinghchhabra** [spring-petclinic/spring-petclinic-microservices#196] [Q&A] [answered]
Can I get source code of @Value internal implement spring

**Comment by arey** [comment, marked as answer]
Do you mean the `@Value`annotation?
Here is the source code: https://github.com/spring-projects/spring-framework/blob/main/spring-beans/src/main/java/org/springframework/beans/factory/annotation/Value.java

**Reply by arvindersinghchhabra** [reply]
Thanks @arey  actually i want to implement the same with custom annotation so that i can execute logic and assign back new value to variable which is annotated with particular field in DTO.
